providers/azure/connection/azureinstancesnapshot/instance.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package azureinstancesnapshot
+
+import (
+	compute "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+	"github.com/cockroachdb/errors"
+	"github.com/rs/zerolog/log"
+)
+
+type instanceInfo struct {
+	subscriptionId string
+	resourceGroup  string
+	instanceName   string
+	location       string
+	bootDiskId     string
+	zones          []*string
+	// Attach the entire VM response as well
+	vm compute.VirtualMachine
+}
+
+func (s *instanceInfo) getFirstAvailableLun() (int, error) {
```

Comment:
we no longer run cmds, we simply use the azure API to tell us what LUN we can attach the disk to